### PR TITLE
Updating container image sections (BSC#1201389, 1201388)

### PR DESCRIPTION
**Pull request by asettle**
Signed-off-by: Alexandra Settle <asettle@suse.com>

### CODE REVIEW

File: xml/admin_ceph_upgrade.xml [hunk 1]
```diff
@@ -389,6 +389,19 @@
    The following procedure describes the process of upgrading the &smaster;:
   </para>
 
+  <important>
+    <para>
+      Ensure the steps have been followed from <xref linkend="deploy-cephadm-configure-imagepath"/>
+      before continuing. Without this configuration, the podman image pull fails
+      within &cephadm;, but will succeed with in terminal if customer is setting
+      following variables:
```
Comment (dariavladykina) [inline xml/admin_ceph_upgrade.xml:397]:
```suggestion
      Before continuing, ensure the steps have been followed 
      from <xref linkend="deploy-cephadm-configure-imagepath"/>. 
      Without this configuration, the podman image pull fails in
      &cephadm;, but will succeed in the terminal if the customer 
      sets the following variables:
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please consider some suggestions here

**Review by tbazant — APPROVED** [review]
See my one note inline

**Review by dariavladykina — COMMENTED** [review]
LGTM

File: xml/admin_ceph_upgrade.xml [hunk 2]
```diff
@@ -389,6 +389,19 @@
...
+    </para>
+<screen>
```
[the remark this answers is not in the document]
Comment (asettle) [reply xml/admin_ceph_upgrade.xml:400]: I believe both

**Pull request merged**